dependent-sum.cabal

```diff
@@ -24,13 +24,7 @@ description:            A dependent sum is a generalization of a
                         dependent sum types by using your own \"tag\"
                         types.
 
-tested-with:            GHC == 7.0.4,
-                        GHC == 7.2.2,
-                        GHC == 7.4.2,
-                        GHC == 7.6.3,
-                        GHC == 7.8.4,
-                        GHC == 7.10.1,
-                        GHC == 7.11
+tested-with:            GHC == 8.4.4
 
 extra-source-files:     examples/*.hs
```

Comment:
The changelog should be listed in the cabal file so that hackage knows about it.